src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer  extends Machine {
+
+    @Override
+    public void doWork() {
+      System.out.println("Bulldozer started working!!!");
+    }
+
+    @Override
+    public void stopWork() {
+      System.out.println("Bulldozer stop working!!!");

```

Comment:
There is a minor grammatical error in the message. It should be "Bulldozer stopped working!!!" instead of "Bulldozer stop working!!!" to maintain consistency with the past tense used in the `doWork` method.